src/Uno.UI-Wasm-only.slnf

```diff
@@ -39,9 +39,8 @@
       "Uno.UI.Toolkit\\Uno.UI.Toolkit.Wasm.csproj",
       "Uno.UI.Wasm.Tests\\Uno.UI.Wasm.Tests.csproj",
       "Uno.UI\\Uno.UI.Wasm.csproj",
-      "Uno.UWPSyncGenerator.Reference\\Uno.UWPSyncGenerator.Reference.csproj",
```

Comment:
It looks like you'll need to rebase your work on top of master, you're not picking up some changes that were done recently.